pkg/handler/handler.go

```diff
@@ -219,6 +222,42 @@ func (h handler) catalog(w http.ResponseWriter, r *http.Request, params map[stri
 	writeDefaultResponse(w, http.StatusOK, resp, err)
 }
 
+func (h handler) getinstance(w http.ResponseWriter, r *http.Request, params map[string]string) {
```

Comment:
nit: `getinstance` -> `getInstance`

Reply:
but it's ugly and isn't all lower like the others in handler :) I really wanted to use get_instance :)